assets/src/edit-story/components/canvas/pagemenu/index.js

```diff
@@ -77,35 +73,30 @@ const Options = styled.div`
 const Divider = styled.span`
   background-color: ${({ theme }) => theme.colors.fg.white};
   opacity: 0.3;
-  height: ${HEIGHT}px;
+  height: ${HEIGHT / 2}px;
```

Comment:
So the icons happen to be half the height of the buttons and the divider should follow the icon height... Is there a cleaner way to do this?

Reply:
Maybe an option could be adding the sizes of the button variants in pixels as well to the `button/index.js` in the design-system folder and using that instead of height? Would be great if we could still access the size in px somehow as well for these cases specifically -- when other things depend on the size, might not be the only time when this happens. Then, inside the `button/index.js` also these constants could be used throughout the file instead of using numbers directly. Thoughts?

Reply:
So, I added the pixel sizes of icon buttons to the button constants in the design system. I'm not loving it, but it works okay. Naming could be optimized.

I'd like input from @BrittanyIRL as well here.

* Adding constants: dda29454e
* Using them: 0fa5c1cc4

Reply:
(also, the icons weren't actually half the size - but slightly smaller than that even: 14px vs 32px button)

Reply:
I feel ok about this because we're already defining the height in pixels here. I would say 1 nit would be can we name the  group `BUTTON_HEIGHT_PIXELS` or something similar that can give us a bit of context as to what the pixels are? 

My assumption reading through this is that the divider would always want to be relative to the button sizes so it makes sense to tie it together.

Reply:
Well, it is both the width and the height of the elements (and in other places we're using it to express width), so that's why the dimension-neutral name. And `SIZE` was already taken.

But it could perhaps be `BUTTON_DIMENSIONS` instead?

Reply:
Changed all this completely, as icons only have one size now. There's a theme constant for the large button size and another for the icon size. That's it.

Reply:
(and neither is used for this particular variable - it's completely custom set to 16px, which looks good, but doesn't directly match the icon heights, as they vary a slight bit)